skaffold.yaml

```diff
@@ -458,6 +458,13 @@ deploy:
           - apply
           - -f
           - example/gardener-local/controlplane/networkpolicy.yaml
+      after:
```

Comment:
Any reason why this is not fine in `before`?

Reply:
It is working. I first thought it has to be `after` because the helm chart is creating `garden` namespace. But I di not notice that the namespace is created in skaffold.yaml right in the beginning.

Reply:
Nice, thank you. I think you also need to adjust https://github.com/gardener/gardener/blob/56361439e51a9048c7f1b57e54fb315db391762d/skaffold.yaml#L494.

Reply:
now, that I see the failing upgrade tests, I remember the reason for `after` 😅 
It has to be `after`, otherwise the upgrade tests fail. In the upgrade test case, helm removes the domain secrets, so they have to be created after helm was running. Otherwise, they are gone.
Thus, I removed the last PR again.